src/ThemedStyleSheet.js

```diff
@@ -59,7 +67,8 @@ export default globalCache.setIfMissingThenGet(
   () => ({
     registerTheme,
     registerInterface,
-    create,
+    create: createLTR,
+    createRTL,
```

Comment:
should we also add `createLTR,` here, for symmetry?

Reply:
I don't know how much sense that makes if `create` always defaults to `createLTR` and I wanna keep create for the default case (and for backwards compatibility)